test/fixtures/mediawiki/valid.vue

```diff
@@ -14,8 +14,11 @@
 			</template>
 		</blah-component>
 		<!-- Valid: vue/html-self-closing -->
-		<p v-i18n-html:foo></p>
 		<blah-component></blah-component>
+		<!-- Valid: vue/no-child-content -->
+		<!-- eslint-disable-next-line vue/no-v-html -->
+		<p v-html="foo"></p>
```

Comment:
This should go in invalid.vue?

Reply:
Confusingly, this is asserting that a tag that uses a directive (`v-html`) and is empty is correct under the `vue/no-child-content` rule. However, there's no way to test that without also running afoul of the `vue/no-v-html` rule. The corresponding case in `invalid.vue` is something like `<p v-html="foo">bar</p>`.

Reply:
Comment this then?